packages/server-boilerplate/src/orchestrator/api/ApiBuilder.ts

```diff
@@ -132,6 +132,18 @@ export class ApiBuilder {
     return this;
   }
 
+  post<T extends ExpressRequest<T> = Request>(
+    path: string,
+    handler: RequestHandler<Params<T>, ResBody<T>, ReqBody<T>, Query<T>>,
+  ) {
+    if (this.verifyAuthMiddleware) {
+      this.app.post(path, this.attachSession, this.verifyAuthMiddleware, handler);
+    } else {
+      this.app.post(path, this.attachSession, handler);
+    }
+    return this;
```

Comment:
I guess it might be nice to consolidate this logic into a single function as it will probably be the same for `PUT` and `DELETE` as well.

Perhaps something like:
```
  get<T extends ExpressRequest<T> = Request>(
    path: string,
    handler: RequestHandler<Params<T>, ResBody<T>, ReqBody<T>, Query<T>>,
  ) {
    return addRoute('get', path, handler);
  }
    
  post<T extends ExpressRequest<T> = Request>(
    path: string,
    handler: RequestHandler<Params<T>, ResBody<T>, ReqBody<T>, Query<T>>,
  ) {
    return this.addRoute('post', path, handler);
  }
    
  addRoute<T extends ExpressRequest<T> = Request>(
    method: 'get' | 'post',
    path: string,
    handler: RequestHandler<Params<T>, ResBody<T>, ReqBody<T>, Query<T>>,
  ) {
    if (this.verifyAuthMiddleware) {
      this.app[method](path, this.attachSession, this.verifyAuthMiddleware, handler);
    } else {
      this.app[method](path, this.attachSession, handler);
    }
    return this;
  }
```